commands/staff/kick.py

```diff
@@ -0,0 +1,24 @@
+import discord
+from discord.ext import commands
+from common.database.decorator import database_connect
+from discord.ext.commands import has_permissions, MissingPermissions
+
+@has_permissions(kick_members=True)
+async def kick(ctx: discord.Interaction, member: discord.Member, reason: str):
```

Comment:
It has no effect on anything, but the common naming convention for the core command function has been name_command. So I personally would name it kick_command, but that's purely presentation and not important.